scss/app.scss

```diff
@@ -1,7 +1,6 @@
 @import 'variables';
 @import 'mixins';
 @import 'colors';
-@import 'fonts';
```

Comment:
Let's not completely remove `scss/_fonts.scss`. There is the potential that we use a font one day that we need to host ourselves, and I want it to be very clear where that should be done.